Author: Alastair McKinstry <mckinstry@debian.org>
Description: Standardize the install of vistrails for packaging. 
 Don't call apt, etc. in distribution packages.
Forwarded: not-needed
Last-Updated: 2014-04-06

Index: vistrails-2.1.1/vistrails/run.py
===================================================================
--- vistrails-2.1.1.orig/vistrails/run.py	2014-04-05 00:17:37.000000000 +0100
+++ vistrails-2.1.1/vistrails/run.py	2014-04-05 02:25:55.061984526 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/python2.7
 ###############################################################################
 ##
 ## Copyright (C) 2011-2014, NYU-Poly.
